packages/fast-components-react-base/src/typography/typography.props.ts

```diff
@@ -14,7 +14,7 @@ export enum TypographyTag {
     figcaption = "figcaption"
 }
 
-export enum TypeLevel {
+export enum Size {
```

Comment:
This should probably be `TypographySize`

Reply:
Fixing throughout